daemon/network.go

```diff
@@ -812,19 +812,19 @@ func buildCreateEndpointOptions(c *container.Container, n *libnetwork.Network, e
 			for _, ips := range ipam.LinkLocalIPs {
 				linkIP := net.ParseIP(ips)
 				if linkIP == nil && ips != "" {
-					return nil, errors.Errorf("Invalid link-local IP address: %s", ipam.LinkLocalIPs)
+					return nil, fmt.Errorf("Invalid link-local IP address: %s", ipam.LinkLocalIPs)
```

Comment:
Are these strings matched anywhere? Would this be the right moment to also fix capitalisation of these errors (there's a couple more `Invalid ....` errors below)?

I see you did some for one error below

```suggestion
					return nil, fmt.Errorf("invalid link-local IP address: %s", ipam.LinkLocalIPs)
```